.github/workflows/unit-test.yml

```diff
@@ -1,13 +1,11 @@
 name: Unit Tests
 
+# Since Unit Tests are required to pass for each PR,
+# we cannot disable them for documentation-only changes.
 on:
   pull_request:
-    paths-ignore:
-    - '**.md'
```

Comment:
I think we should explore ways to allow this kind of config using `if` statements in the jobs. I think some reusable github actions already exist for this use-case.

It's also very useful if we decide one day to consolidate in a single workflow.

Reply:
See #28669 for an exploration and some discussion :slightly_smiling_face: